releasenotes/notes/0.45/dag-appenders-check-84d4ef20c1e20fd0.yaml

```
@@ -1,5 +1,5 @@
 ---
-features:
+features_circuits:
```

We normally view the dagcircuit as a transpiler feature and not a circuit feature (the dagcircuit is primarily the ir for the transpiler).

```suggestion
features_transpiler:
```